src/SUMMARY.md

```diff
@@ -21,3 +21,11 @@
 - [Trait Solver 5](./trait_solver/5.md)
 - [Trait Solver 6](./trait_solver/6.md)
 - [Trait Solver 7](./trait_solver/7.md)
+
+# Borrowck
```

Comment:
can we write out the ck and call it "borrow checker"?

Reply:
I agree